### PR TITLE
Navigation and misc fixes

### PR DESCRIPTION
**Pull request by andreas-bergstrom**
- Location of code examples improved
- Headlines formatting improved
- Added some CSS and JS to have one more navigation level in the ToC-column to the left (see picture)

Screenshot 2019-11-28 at 14 21 13

### CODE REVIEW

File: source/api-push.html.md [hunk 1]
```diff
@@ -97,7 +48,7 @@ export APP_ID='0123456789123456'
 export APP_KEY='AbC123XyZ'
 ```
 
-### Basic example
+> Basic example:
```
Comment (2019342a) [inline source/api-push.html.md]: Why `>`?

File: source/api-push.html.md [hunk 2]
```diff
@@ -396,7 +391,7 @@ Status Code: 2XX
 | owner          | object  | The [user](#user) that owns the application                                   |
 | ab_test        | string  | A unique identifier used for A/B testing                                      |
 
-##### Source Type
+### Source Type
```
Comment (2019342a) [inline source/api-push.html.md]: Why lower?

File: source/api-push.html.md (hunk 1 above)
Comment (andreas-bergstrom) [reply source/api-push.html.md]: To move it to the rightmost codeexample-column:
https://github.com/slatedocs/slate/wiki/Markdown-Syntax

File: source/api-push.html.md (hunk 2 above)
Comment (andreas-bergstrom) [reply source/api-push.html.md]: Just a leftover from when I copied it from Github Wiki, the ##### made it too small and at ### it's also included in the leftmost ToC.

**Pull request merged**